src/app/api/public/contracts/[contractName]/[usersContractVersion]/[functionName]/route.ts

```diff
@@ -106,6 +107,24 @@ async function postHandler(req: NextRequestWithAuth, { params: { contractName, u
 
       const transactionResult = await gaslessTransaction(account, calldata);
 
+      const txRes = await provider.waitForTransaction(transactionResult.transactionHash) as any;
+      console.log("Interaction response: ",txRes)
+      const fee = parseInt((txRes as any)?.actual_fee?.amount, 16);
```

Comment:
is it correctly formatting fees to wei? asking becasue i have no idea and want to double-check with you

Reply:
Yes